dpctl/tensor/_clip.py

```diff
@@ -405,11 +402,9 @@ def clip(x, min=None, max=None, out=None, order="K"):
         min ({None, usm_ndarray}, optional): Array containing minimum values.
```

Comment:
Should we document that `min` and `max` can actually be Python scalars?

```

In [1]: import dpctl.tensor as dpt, matmul as mm, dpctl

In [2]: x = dpt.remainder(dpt.arange(10**6, dtype="i4"), 171)

In [3]: y = dpt.clip(x, 0, 5)

In [4]: dpt.min(y), dpt.max(y)
Out[4]: (usm_ndarray(0, dtype=int32), usm_ndarray(5, dtype=int32))
```

Reply:
Yes, we definitely should, good catch